beta/src/pages/learn/index.md

```diff
@@ -23,7 +23,7 @@ Welcome to the React documentation! Here is an overview of what you can find on
 
 This is a tiny React app. To get your first taste of React, **edit the code below** and make it display your name:
 
-<Sandpack>
+<Sandpack showDevTools>
```

Comment:
This was good for demonstrating the new functionality but let's remove it before landing so that nothing changes as the result of this PR. Then we (@gaearon, @rachelnabors, and I) can add it where it makes sense in follow up PRs.